File: node/hello-http.js
var http = require('http');

const PORT = 8080;
var server = http.createServer((req, res) => {
  res.writeHead(200, { 'Content-Type': 'text/plain' });
  res.write("Hello World!!\n");
  res.end;
}).listen(PORT);

console.log("Server is running on http://localhost:%d", PORT);

File: dart/hello-http.dart
import 'dart:io';

void main(List<String> args) {
  HttpServer.bind(InternetAddress.loopbackIPv4, 8080).then((server) {
    print("Server is listening on http://${server.address.host}:${server.port}");

    server.listen((HttpRequest req) {
      req.response.write('Hello, world!');
      req.response.close();
    });
  });
}
